packages/node_modules/@webex/internal-plugin-conversation/src/decryption-transforms.js

```diff
@@ -126,6 +126,9 @@ export const transforms = toArray('inbound', {
     if (conversation.avatarEncryptionKeyUrl) {
       promises.push(ctx.transform('decryptObject', conversation.avatarEncryptionKeyUrl, conversation.avatar));
     }
+    if (conversation.previous) {
+      promises.push(ctx.transform('decryptPropDisplayName', usableKey, conversation.previous));
+    }
```

Comment:
I'm confused why we're doing this in decryptConversation and not decryptActivity

Reply:
^^discussed offline